### PR TITLE
[2.13.x] DDF-3941 Whitelisted WSDL URLs

### PR DESCRIPTION
**Pull request by blen-desta**
#### What does this PR do?
- Whitelisted WSDL URLs
- Whitelisted POST to `/services/login`

#### Who is reviewing it? 
@garrettfreibott @brjeter 

#### Ask 2 committers to review/merge the PR and tag them here.
@coyotesqrl
@stustison

#### How should this be tested?
- Able to access the wsdl urls listed under `/services` 
- Able to log in by going to `/login`

#### What are the relevant tickets?
[DDF-3941](https://codice.atlassian.net/browse/DDF-3941)

#### Checklist:
- [ ] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/security/filter/security-filter-csrf/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 1]
```diff
@@ -20,6 +19,7 @@
         <property name="whiteListContexts">
             <array value-type="java.lang.String">
                 <value>/services[/]?$=GET</value>
+                <value>/services/.*\?wsdl$=GET</value>
```
Comment (coyotesqrl) [inline platform/security/filter/security-filter-csrf/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: @stustison Would we need to whitelist both the question mark and dot wsdl paths?

Comment (blen-desta) [reply platform/security/filter/security-filter-csrf/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: I changed that to check the queryString.

**Comment by blen-desta** [comment]
build now

**Comment by coyotesqrl** [comment]
It still looks like it might have a problem with `.wsdl` lookups.

File: platform/security/filter/security-filter-csrf/src/main/resources/OSGI-INF/blueprint/blueprint.xml (hunk 1 above)
Comment (stustison) [reply platform/security/filter/security-filter-csrf/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: No, we don't expose anything via ".wsdl" (which is incorrect anyways). There were only some extremely old (read: legacy) systems that did it the wrong way.

**Comment by blen-desta** [comment]
build now

**Pull request merged**